rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] The rental should add itself to the @person the 'belongs-to' association, It will look something like this :point_down: 


```ruby
    person.rentals << self
```